### DISCUSSION TITLE
Machine Learning Concept

### DISCUSSION THREAD

**Discussion by FazlullahBokhari** [community/community#18163] [Discussions] [answered]
## Question Of the day
1.What is Machine Learning Pipeline?

**Comment by david-s-martinez** [comment, marked as answer]
Well, the answer is in the name. A pipeline network defines the flow of a fluid. In this case the fluid would be the data and the pipeline would be the series of methods, programs or systems that are used to ensure the flow of such data and could involve data mining, data collection, data processing, training, validation and deployment for machine learning algorithms. For example, an autonomous car needs a machine learning pipeline to train a model to drive itself. The input data could be images from cameras or  raw data from sensors. Such data would have to go through a machine learning pipeline to be useful to train a deployable neural network in charge of driving the car.

**Comment by MarketingPip** [comment]
Hello @fazlullahb0 , 

I just posted this on another user's post but it appear's you may have mistaken this community for a general discussions area as well. Keep in mind tho this repository & it's discussions area are NOT meant for this purpose.  The purpose of these discussions are as following text states - 


           These discussions are where you can share suggestions for how the products should be improved and discuss

          those improvements with the community, including members of the GitHub product team. Check out [Making suggestions]

          (https://github.com/github-community/community#making-suggestions) to learn how to provide feedback.



By posting things like this - it takes time out of the day for the @github-staff which they try to sort through / read. Which results in thing's taking longer to get improved for us on [GitHub](https://github.com)  as it makes it harder for them to read & find  important suggestions. And valuable feedback that help's improve the site! 

There is more info on the guidelines for using this repo & discussion's properly in the README which you can find at the link below / or in the README of this repo. 

[Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more!](https://github.com/github-community/community)

I would assume this was a general mistake but - just posting this so this thread doesn't follow with any other replies!
:slightly_smiling_face: 

I maybe wrong - but I would assume they would appreciate them deleting this if possible! 

ps - tagging @MrKrishnaAgarwal @davidmartinez13 to give the heads up to both as well!

**Comment by FazlullahBokhari** [comment]
Hi @MarketingPip 
I have no idea how to delete it.

Can you please delete these dissecussion from you end so that there should not be be conflicts. 


Thank you

**Reply by MarketingPip** [reply]
@fazlullahb0 I am not a staff member or affiliated with GitHub in anyway - just popped in here to post a suggestion for GitHub and seen a few of these posts which confused me until I read through the README. 

Just wanted to give you a heads up & other member's posting in here the same. I would assume they will delete it shortly. 

But you are welcome! :+1:

**Reply by MrKrishnaAgarwal** [reply]
Sure, Thanks I have deleted my reply @MarketingPip
